app/api/repositories/event_repository.py

```diff
@@ -106,6 +106,35 @@ async def add_guest_to_event(
 
         return res
 
+    async def add_organizer_to_event(
+        self,
+        event_id: PydanticObjectId,
+        organizer_id: PydanticObjectId,
+        session: AsyncIOMotorClientSession | None = None,
```

Comment:
can use Optional from typing

Reply:
Fixed in [commit](https://github.com/liel-almog/gala/pull/9/commits/2dbb70d0ea89d2ea44b2039cd1c82616a3b7ea7a)